transaction/std/src/memo_builder/gift_code_sender_memo_builder.rs

```diff
@@ -171,9 +197,26 @@ mod tests {
     #[test]
     fn test_gift_code_sender_note_builder_creation_fails_with_invalid_note() {
         // Create Memo Builder with an input longer than allowed
-        let note_bytes = [b'6'; GiftCodeSenderMemo::MEMO_DATA_LEN + 1];
+        let note_bytes = [b'6'; GiftCodeSenderMemo::NOTE_DATA_LEN + 1];
         let note = std::str::from_utf8(&note_bytes).unwrap();
         let builder = GiftCodeSenderMemoBuilder::new(note);
         assert!(matches!(builder, Err(NewMemoError::BadInputs(_))));
     }
+
+    #[test]
+    fn test_gift_code_sender_memo_builder_fee_token_cannot_be_different_from_change_token() {
```

Comment:
similar to other builders consider a test case for default fee

Reply:
Same as above, default fee is never used memo builders